### DISCUSSION TITLE
Unable to modify bookmark name

### DISCUSSION THREAD

**Discussion by leewinq** [sumatrapdfreader/sumatrapdf#2674] [General]
Unable to modify bookmark name

**Comment by GitHubRulesOK** [comment]
Bookmarks are provided by document author
so to modify you need an authors editor application like Adobe Pro or Microsoft Office
or one of thousands of document editors
![image](https://user-images.githubusercontent.com/12754913/197357664-89cc8cb8-9fad-40d6-a689-b5538b72c8b4.png)

**Reply by abelokoj-github** [reply]
I don't think this is the case. You can make use of the free Foxit pdf reader to edit bookmarks. It is a feature that will be highly appreciated.

**Reply by GitHubRulesOK** [reply]
Chapter 8: Advanced ***Editing*** on PDFs
Size: 559 MB (586,782,782 bytes)
1,036 Files, 117 Folders

**Reply by abelokoj-github** [reply]
This must be referring to editing the pdf document itself (this is not allowed in the Foxit pdf reader free version) however, editing bookmarks is allowed.

Here is an example below You can download Foxit pdf reader and see it for yourself. Just open any pdf and press **CTRL+B** it will give you an option to add a bookmark to the current page

There is also another free app that allows editing bookmarks (though a command line tool).  Here is the link to their page https://community.coherentpdf.com/

But this also points again that in can be done in Sumatra pdf (unless there are strong reasons that prevents it from being implemented).  see pictures below

![image](https://user-images.githubusercontent.com/69094732/197354928-51a04ac8-d003-4c4e-9c5b-0889926fee67.png)

![image](https://user-images.githubusercontent.com/69094732/197355099-2b2f7af8-d9a1-4223-aa1f-1b537aa74797.png)

**Comment by abelokoj-github** [comment]
I think this feature has long been in existence far before Foxit pdf version 12.  I have been using Foxit for a long time and it has been there.

See the picture below which dates back to the Foxit version 4.3. Version 4.3 has been released since far back in Nov 2010.  see link here
[http://cdn01.foxitsoftware.com/pub/foxit/manual/en_us/FoxitReader61_Manual.pdf](url) 
[https://www.foxit.com/pdf-reader/version-history.html](url)

Foxit version 4.3 has been released since far back Nov 2010 with the included bookmark editing feature in the free version.  _**I honestly believe that  Sumatra pdf can have the same bookmark editing feature. Though it might be hard work on the part of developers (which is quite understandable)**_ But I strongly believe it can be done. Unfortunately, I am not an app developer, would have loved to make necessary contributions to see this come true.


![image](https://user-images.githubusercontent.com/69094732/197361782-0a59f855-1686-4520-895c-c13edf9b1fc3.png)



![image](https://user-images.githubusercontent.com/69094732/197361894-f5f763d9-a12f-417a-b5c2-f2cd4972c1d5.png)
